config/lib/express.js

```diff
@@ -141,7 +141,7 @@ module.exports.initModulesConfiguration = function (app, db) {
  */
 module.exports.initHelmetHeaders = function (app) {
   // Use helmet to secure Express headers
-  var SIX_MONTHS = 15778476000;
+  var SIX_MONTHS = 15778476;
```

Comment:
Would it make sense to just use `15778476` directly in hsts settings object since this variable isn't used anywhere else?